pkg/kube/wait_test.go

```diff
@@ -266,6 +266,16 @@ func Test_waiter_jobReady(t *testing.T) {
 			args: args{job: newJob("foo", 1, 1, 0, 1)},
 			want: false,
 		},
+		{
+			name: "job is completed with backoffLimit=0",
+			args: args{job: newJob("foo", 0, 1, 1, 0)},
+			want: true,
+		},
+		{
+			name: "job is failed and it is ok",
+			args: args{job: newJob("foo", 0, 1, 0, 1)},
+			want: true,
```

Comment:
Hi @aCLr - can you verify if this PR solves the same issue? https://github.com/helm/helm/pull/9313

I noticed this `want: true` is the opposite as the test provided in that PR?

Reply:
No this is not the same. My PR closes #9313 while #9285 can't do the same.
Currently we use that patch in production and everything works like a charm.

Reply:
@aCLr - sorry can you clarify? You use _this_ patch, or the one provided in #9285 ?